iota/commands/extended/get_transaction_objects.py

```diff
@@ -0,0 +1,59 @@
+# coding=utf-8
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from typing import Iterable, List, Optional
+
+import filters as f
+
+from iota import Transaction, TransactionHash
+from iota.commands.core import GetTrytesCommand
+from iota.commands import FilterCommand, RequestFilter
+from iota.filters import Trytes
+
+__all__ = [
+    'GetTransactionObjectsCommand',
+]
+
+
+class GetTransactionObjectsCommand(FilterCommand):
+    """
+    Executes `GetTransactionObjectsCommand` command.
+
+    See :py:meth:`iota.api.StrictIota.get_transaction_objects`.
+    """
+    command = 'getTransactionObjects'
+
+    def get_request_filter(self):
+        return GetTransactionObjectsRequestFilter()
+
+    def get_response_filter(self):
+        pass
+
+    def _execute(self, request):
+        hashes = request\
+            .get('hashes') # type: Optional[Iterable[TransactionHash]]
```

Comment:
```suggestion
            .get('hashes') # type: Iterable[TransactionHash]
```

(the `f.Required` filter in the request filter chain ensures that this value cannot be `None`)